R/response_sources.R

```diff
@@ -136,11 +136,11 @@ bor_pd <- event(
 #' @export
 bor_ne <- event(
```

Comment:
shouldn't this update be for cbor_ne rather than bor_ne? that's what we did for admiralroche.

Reply:
We do not have `cbor_ne` in admiralonco. We use `bor_ne` for both BOR and CBOR.

Reply:
Do you think we should change this?

Reply:
ok, happy to go with your call as I see similar case with bor_sd. i'll approve for now and then if you decide to update i can re-review.

Reply:
we should also update ibor_ne for iRECIST

Reply:
Good point.
Updated